fixtures/rabota_fixtures.py

```diff
@@ -0,0 +1,231 @@
+
+
+@pytest.fixture
+def get_response_from_wrong_url():
+    """
+    Getting response for incorrect url
+    """
+    headers = {
+        'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/webkit-version (KHTML, like Gecko) '
+                      'Silk / browser - version like Chrome / chrome - version Safari / webkit - version'}
+    wrong_url = 'https://rabota.by/search/vacncy?clusters=true&' \
+                'area=16&enable_snippets=true&salary=&st=searchVacancy&text=python'
+    return HTTPClient.get(wrong_url, headers=headers)
+
+
+@pytest.fixture
+def get_query_shotgun():
+    """
+    Getting response for 'shotgun' query
+    """
+    headers = {
+        'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/webkit-version (KHTML, like Gecko) '
+                      'Silk / browser - version like Chrome / chrome - version Safari / webkit - version'}
+    url_shotgun = 'https://rabota.by/search/' \
+                  'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=shotgun'
+    return HTTPClient.get(url_shotgun, headers=headers)
+
+
+@pytest.fixture
+def id_for_first_vacancy(get_response):
+    """
+    Returns id for first vacancy
+    """
+    lxml_text = RabotaByParser.get_lxml(get_response.text)
+    list_id = RabotaByParser.get_list_of_vacancy_id(lxml_text)
+    return list_id[0]
+
+
+@pytest.fixture
+def get_response_for_first_vacancy_by_id(id_for_first_vacancy):
+    """
+    Getting response for first vacancy of 'python' query
+    """
+    headers = {
+        'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/webkit-version (KHTML, like Gecko) '
+                      'Silk / browser - version like Chrome / chrome - version Safari / webkit - version'}
+    return HTTPClient.get(f'https://rabota.by/vacancy/{id_for_first_vacancy}', headers=headers)
+
+
+@pytest.fixture
+def description_for_first_vacancy(get_response_for_first_vacancy_by_id):
+    """
+    Returns description for first vacancy in query 'python'
+    """
+    return RabotaByParser.find_vacancy_description(get_response_for_first_vacancy_by_id.text).text.lower()
+
+
+@pytest.fixture
+def occur_python_on_first_vacancy(description_for_first_vacancy):
+    """
+    Returns amount of occurrences for the word 'python' on the first vacancy
+    """
+    return RabotaByParser.count_word('python', description_for_first_vacancy)
+
+
+@pytest.fixture
+def occur_linux_on_first_vacancy(description_for_first_vacancy):
+    """
+    Returns amount of occurrences for the word 'linux' on the first vacancy
+    """
+    return RabotaByParser.count_word('linux', description_for_first_vacancy)
+
+
+@pytest.fixture
+def occur_flask_on_first_vacancy(description_for_first_vacancy):
+    """
+    Returns amount of occurrences for the word 'flask' on the first vacancy
+    """
+    return RabotaByParser.count_word('flask', description_for_first_vacancy)
+
+
+@pytest.fixture
+def avg_occur_for_python(get_response):
```

Comment:
avg_occur_for_python , avg_occur_for_linux and avg_occur_for_flask are duplicates.
Create one fixture which could work with any word which comes as parameter